### ISSUE TITLE
README example is excessively large

### ISSUE THREAD

**Issue by glwagner** [CliMA/Oceananigans.jl#961]
The README example builds a grid of size (100, 100, 50), which is non-small. We should make the intro example more bite-size.

**Comment by navidcy** [comment]
my 2 cents: It's a bit "tricky" to have code in the README since with constant code upgrades oftentimes the README code gets outdated...

**Comment by glwagner** [comment]
> my 2 cents: It's a bit "tricky" to have code in the README since with constant code upgrades oftentimes the README code gets outdated...

That's true, probably best not to have too much. I personally find code in the README to be super helpful for "quick starting" though...

**Comment by navidcy** [comment]
Actually me too!
(unless it's broken)

**Comment by glwagner** [comment]
Actually, it probably wouldn't be that hard to run code in the README in a test. Somebody should write a package for that.

**Comment by ali-ramadhan** [comment]
Could do something like
```julia
using Markdown
readme = Markdown.parse(read("README.md", String))
julia_code_snippets = filter(c -> c isa Markdown.Code && c.language == "julia", readme.content)
for code_snippet in julia_code_snippets
    Meta.parse("begin $(code_snippet.code) end") |> eval
end
```
but then the code blocks under "Installation instructions" should not be `julia` code blocks.

**Comment by glwagner** [comment]
> but then the code blocks under "Installation instructions" should not be julia code blocks.

That's ok for me... we really want "julia-repl" highlighting for those blocks anyways.

Or, we can filter the lines that start with `julia>` ?

**Comment by glwagner** [comment]
It doesn't actually help to ask for julia syntax highlighting in the installation instructions anyways:

![image](https://user-images.githubusercontent.com/15271942/93905610-281f7e00-fcc9-11ea-93ed-6bb87823aae1.png)

**Comment by glwagner** [comment]
What it looks like for me:

![image](https://user-images.githubusercontent.com/15271942/93905734-44bbb600-fcc9-11ea-83be-644d9202e591.png)

In fact, a screenshot rather than code could make more sense ?

**Comment by glwagner** [comment]
Actually wondering if we should provide code that installs julia:

```julia
using Pkg
Pkg.add("Oceananigans")
Pkg.instantiate()
```

Then people can copy paste like they really want to. 

If they know how to use the package manager, they already know what to do (or we can add a separate comment for such people).

**Comment by ali-ramadhan** [comment]
Hmmm, yeah copy-pastable code for installation might be better. Would rather use package mode but the benefit of being to copy paste is pretty sweet.

**Comment by glwagner** [comment]
We could use something like

> Julia's package manager will happily install the latest version of Oceananigans for you:
> 
> ```julia
> using Pkg
> Pkg.add("Oceananigans")
> Pkg.instantiate()
> ```
> 
> Alternatively, you can press `]` from Julia's REPL to enter package manager mode and type `pkg> add Oceananigans` followed by `pkg> instantiate`. We recommend installing Oceananigans with the built-in Julia package manager, because this installs a stable, tagged release.

**Comment by glwagner** [comment]
Another README issue: we should change

> It is designed to solve the rotating Boussinesq equations...

to 

> It solves the rotating Boussinesq equations...

**Comment by glwagner** [comment]
I'm closing this issue because I'm judging that it's not of current, timely relevance to Oceananigans development. If you would like to make it a higher priority or if you think the issue was closed in error please feel free to re-open.
